vars/sonarExecuteScan.groovy

```diff
@@ -191,3 +206,30 @@ private void loadSonarScanner(config){
         mv ${foldername} .sonar-scanner
     """
 }
+
+private void loadCertificates(Map config) {
+    String certificateFolder = '.certificates/'
+    List wgetOptions = [
+        "--directory-prefix ${certificateFolder}"
+    ]
+    List keytoolOptions = [
+        '-import',
+        '-noprompt',
+        '-storepass changeit',
```

Comment:
we expect the default password?

Reply:
I'll extract this to the config with `changeit` as default, what do you think?

Reply:
sounds reasonable

Reply:
Actually, the keystore is provided in the SonarScanner download. Therefore I think it's ok to make this assumption.